spec/models/replication_transfer_spec.rb

```diff
@@ -35,6 +35,36 @@
       expect(r.update(stored: true)).to be true
       expect(r.bag.replicating_nodes).to include r.to_node
     end
+
+    context "when fixity_value set" do
+      let(:correct_fixity) { "98734723942304820348203840238402983409283408234" }
+      let(:fixity_alg) { Fabricate(:fixity_alg) }
+      let(:bag) {
+        bag = Fabricate(:bag_without_digests)
+        bag.message_digests << Fabricate.build(:message_digest, fixity_alg: fixity_alg, value: correct_fixity)
+        bag.save!
+        bag
+      }
```

Comment:
On a multi-line `let`, use the `begin ... end` block idiom rather than `{ ... }`, e.g.
```ruby
let(:bag) begin
  # ...
end
```